### ISSUE TITLE
python3 fails to build on Linux

### ISSUE THREAD

**Issue by jackgerrits** [microsoft/vcpkg#4603]
It looks like boost-python is trying to invoke msbuild against a vcxproj on Linux (Ubuntu WSL)

```
14:40:57 vcpkg_linux master  > ./vcpkg install boost-python:x64-linux
The following packages will be built and installed:
  * boost-bimap[core]:x64-linux
  * boost-format[core]:x64-linux
  * boost-graph[core]:x64-linux
  * boost-multi-index[core]:x64-linux
  * boost-parameter[core]:x64-linux
  * boost-property-map[core]:x64-linux
  * boost-property-tree[core]:x64-linux
    boost-python[core]:x64-linux
  * boost-xpressive[core]:x64-linux
  * python3[core]:x64-linux
Additional packages (*) will be modified to complete this operation.
Starting package 1/10: python3:x64-linux
Building package python3[core]:x64-linux...
-- Using cached /mnt/c/w/repos/vcpkg_linux/downloads/python-cpython-v3.6.4.tar.gz
-- Extracting source /mnt/c/w/repos/vcpkg_linux/downloads/python-cpython-v3.6.4.tar.gz


-- Applying patch /mnt/c/w/repos/vcpkg_linux/ports/python3/0004-Fix-iomodule-for-RS4-SDK.patch
-- Applying patch /mnt/c/w/repos/vcpkg_linux/ports/python3/0005-Fix-DefaultWindowsSDKVersion.patch
-- Using source at /mnt/c/w/repos/vcpkg_linux/buildtrees/python3/src/v3.6.4-5bfb0301b2
-- Applying patch /mnt/c/w/repos/vcpkg_linux/ports/python3/0001-Static-library.patch
-- Building /mnt/c/w/repos/vcpkg_linux/buildtrees/python3/src/v3.6.4-5bfb0301b2-Lib-static-crt-dynamic/PCBuild/pythoncore.vcxproj for Release
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:56 (message):
    Command failed: msbuild;/mnt/c/w/repos/vcpkg_linux/buildtrees/python3/src/v3.6.4-5bfb0301b2-Lib-static-crt-dynamic/PCBuild/pythoncore.vcxproj;/p:Configuration=Release;/t:Rebuild;/p:Platform=x64;/p:PlatformToolset=external;/p:VCPkgLocalAppDataDisabled=true;/p:UseIntelMKL=No;/p:WindowsTargetPlatformVersion=;/m;/p:WholeProgramOptimization=false
    Working Directory: /mnt/c/w/repos/vcpkg_linux/buildtrees/python3/x64-linux-rel
    See logs for more information:

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_msbuild.cmake:119 (vcpkg_execute_required_process)
  ports/python3/portfile.cmake:55 (vcpkg_build_msbuild)
  scripts/ports.cmake:71 (include)


Error: Building package python3:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: python3:x64-linux
  Vcpkg version: 2018.10.20-unknownhash

Additionally, attach any relevant sections from the log files above.
```

**Comment by luzpaz** [comment]
#5059

**Comment by wangli28** [comment]
Thanks for posting this issue. Closed as duplicate of #4135.
